source/main_module/car/car.c

```diff
@@ -70,21 +73,21 @@ void carPeriodic()
     /* Set Default Outputs */
     // Set Outputs that are purely combinational (do not depend on previous events)
     // Default values will typically reflect those in the IDLE state
-    car.torque_r.torque_left  = 0.0f;
-    car.torque_r.torque_right = 0.0f;
-    car.buzzer    = false;
-    car.sdc_close = true;
+    // car.torque_r.torque_left  = 0.0f;
+    // car.torque_r.torque_right = 0.0f;
+    // car.buzzer    = false;
+    // car.sdc_close = true;
```

Comment:
We can’t forget to uncomment the initialization code

Reply:
Yeah lol that is getting updated when I finish car.c